.gitpod.yml

```diff
@@ -31,25 +21,13 @@ tasks:
       gp open increment/src/lib.rs
       gp open increment/src/test.rs
       gp open README.md
-      soroban contract invoke --id 1 --wasm increment/target/wasm32-unknown-unknown/release/soroban_increment_contract.wasm -- increment
+      stellar contract inspect --wasm increment/target/wasm32-unknown-unknown/release/soroban_increment_contract.wasm
 vscode:
   extensions:
     - rust-lang.rust-analyzer
     - vadimcn.vscode-lldb
 
-github:
```

Comment:
The configuration has been deprecated.
https://www.gitpod.io/changelog/activity-based-prebuilds